### PR TITLE
TxSender JSON Switch Removed

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
image

### CODE REVIEW

**Review by vuvoth — COMMENTED** [review]
Look good! However you should remove unused functions e.g `convertObjectValuesToUnit8Array`
isJson in TxSender.vue

**Comment by Megha-Dev-19** [comment]
> Look good! However you should remove unused functions e.g `convertObjectValuesToUnit8Array` isJson in TxSender.vue

oh yes, thanks for informing, done

File: src/pages/TxSender.vue [hunk 1]
```diff
@@ -142,31 +129,18 @@ export default defineComponent({
...
-				) as EncodedSignedTransaction;
+			const decodedTxn = decodeObj(
+				convertBase64ToUnit8Array(this.txInput)
+			) as EncodedSignedTransaction;
 
-				txID = algosdk.Transaction.from_obj_for_encoding(decodedTxn.txn).txID();
-			} else {
-				// TODO : ADD TxID support for json
-			}
+			txID = algosdk.Transaction.from_obj_for_encoding(decodedTxn.txn).txID();
```
Comment (vuvoth) [inline src/pages/TxSender.vue]: You can use const at here!

**Review by vuvoth — APPROVED** [review]
LGTM! Please format code before merge!

File: src/pages/TxSender.vue (hunk 1 above)
Comment (Megha-Dev-19) [reply src/pages/TxSender.vue]: oops, will update

**Pull request merged**